src/views/Generator/Generator.tsx

```diff
@@ -125,7 +128,11 @@ export function Generator() {
   return (
     <View
       title="Generator"
-      subTitle={getFileNameWithoutExtension(fileName)}
+      subTitle={
+        <>
+          {getFileNameWithoutExtension(fileName)} {isDirty && '(edited)'}
```

Comment:
I considered using icons instead of appending `(edited)` to the title but I didn't think it was as clear as this. Open to suggestions.

<img width="442" alt="image" src="https://github.com/user-attachments/assets/508eb6e8-7ed7-413c-8a4f-4e88fcf2a8a0" />

Reply:
I ended up replacing it with `*` for simplicity.